### ISSUE TITLE
Does not work in qBittorrent v5.0.0

### ISSUE THREAD

**Issue by FireBoy00** [witalihirsch/qBitTorrent-fluent-theme#62]
The progress bars are red and not blue. Although I like them more in red, it looks like it has some sort of error.
![image](https://github.com/user-attachments/assets/dc2cf967-7fcd-4ee4-9b1f-4af8fae76668)

The green line that shows the progress is no longer there.
![image](https://github.com/user-attachments/assets/654dbab3-111f-49d8-b53a-515bb6740b8b)

Those are the issues I noticed, I'll come back or make another issue if I notice any other.

**Comment by tuongdai252** [comment]
The progress bars are brown for me. It might follow accent color? 
(Mostly comment to follow when the green line is fixed).

**Comment by FireBoy00** [comment]
In that case I'd like it if they either don't follow the Windows accent color or just have some sort of way to choose if you wanna have accent color.

By the way, you can follow a GitHub issue even without posting a comment. I don't mean to be rude by saying this, but I just thought I'd let you know.🙃

**Comment by tuongdai252** [comment]
> By the way, you can follow a GitHub issue even without posting a comment. I don't mean to be rude by saying this, but I just thought I'd let you know.🙃

It's okay 😉 Thanks for letting me know. I didn't notice GitHub has had a subscribe button.

**Comment by ArtexJay** [comment]
> In that case I'd like it if they either don't follow the Windows accent color or just have some sort of way to choose if you wanna have accent color.

Make an issue on the official Qbittorrent github page, they should add a toggle for it in the settings.

**Comment by oomek** [comment]
I'm hoping this repo is still maintained. I really like this theme, but the missing progress bars in qBT 5.0.0 is a deal breaker for me.

**Comment by philkonick** [comment]
I can confirm that the Progress and Availability bars use your Windows accent colour (at least for Win 11). If you change your accent colour the bars will recolour after restarting qBittorrent.

**Comment by philkonick** [comment]
> @philkonick when i follow what you suggested, it shows up for a second and then it disappears again. win 10.

Just to clarify, I was referring to the bars in the first screenshot in the initial post, not the download progress bars in the torrents table. Those are not showing at all for me, just like the second screenshot in the initial post. Not sure if that was clear in my last comment.

**Comment by coldfire7** [comment]
![2024-10-14_02-14-57](https://github.com/user-attachments/assets/fc7c18d8-ed94-439d-afc6-ccf2e2886fb3)
Pretty sure it's not supposed to do that...

**Comment by JurecUA** [comment]
![image](https://github.com/user-attachments/assets/a172e31e-ab0b-4676-b9ac-7a3a0c570f59)

**Comment by oomek** [comment]
Some brave soul has just opened a pull request with a fix!

**Comment by skirol** [comment]
> Some brave soul has just opened a pull request with a fix!

no release yet?

**Comment by tuongdai252** [comment]
> > Some brave soul has just opened a pull request with a fix!
> 
> no release yet?
Those 4 qbtheme files are in here:
https://github.com/witalihirsch/qBitTorrent-fluent-theme/tree/main/Builds
No version for default icons tho.

**Comment by skirol** [comment]
> > > Some brave soul has just opened a pull request with a fix!
> > 
> > 
> > no release yet?
> 
> Those 4 qbtheme files are in here: https://github.com/witalihirsch/qBitTorrent-fluent-theme/tree/main/themes
> 
> ~No version for default icons tho.~ Nvm. I downloaded the code and was able to compile the defaulticons versions.

yeah i prefer old icons too. can you please upload the qbtheme file with old icons somewhere? is that a possibility?

**Comment by tuongdai252** [comment]
> yeah i prefer old icons too. can you please upload the qbtheme file with old icons somewhere? is that a possibility?

[defaulticons-fluent.zip](https://github.com/user-attachments/files/17586876/defaulticons-fluent.zip)

**Comment by skirol** [comment]
> > yeah i prefer old icons too. can you please upload the qbtheme file with old icons somewhere? is that a possibility?
> 
> [defaulticons-fluent.zip](https://github.com/user-attachments/files/17586876/defaulticons-fluent.zip)

Thank you! Although I thought you kept the old icons from the fluent theme, not the default qbittorrent icons. Like these: 
![Clipboard-66](https://github.com/user-attachments/assets/167647c9-1e15-4908-8192-f9b2da8685b3)

**Comment by tuongdai252** [comment]
> Thank you! Although I thought you kept the old icons from the fluent theme, not the default qbittorrent icons. Like these: !

Ah. You meant the icons before the update with qbittorrent  5.0.
[Pre-5.0-icons-fluent.zip](https://github.com/user-attachments/files/17588035/Pre-5.0-icons-fluent.zip)

**Comment by skirol** [comment]
> > Thank you! Although I thought you kept the old icons from the fluent theme, not the default qbittorrent icons. Like these: !
> 
> Ah. You meant the icons before the update with qbittorrent 5.0. [Pre-5.0-icons-fluent.zip](https://github.com/user-attachments/files/17588035/Pre-5.0-icons-fluent.zip)

Yes! Thank you so much for doing this for me.

**Comment by PedjaSenpai** [comment]
> > yeah i prefer old icons too. can you please upload the qbtheme file with old icons somewhere? is that a possibility?
> 
> [defaulticons-fluent.zip](https://github.com/user-attachments/files/17586876/defaulticons-fluent.zip)

Is this same as defaulticons-fluent-dark-no-mica ? since among those 4 theme files i cant seem to find one with default icons

**Comment by perigozin** [comment]
The Mica effect isn't working in this version from the pull request. Also the alternative tray icon isn't available anymore.
